app/components/paperShow/components/plutoBlogPosting.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import axios from "axios";
+import { withStyles } from "../../../helpers/withStylesHelper";
+const styles = require("./plutoBlogPosting.scss");
+
+const BLOG_SCRIBER_API_HOST = "https://7hnqfzk1r6.execute-api.us-east-1.amazonaws.com/prod/blogLinks";
+
+interface BlogLink {
+  id: string;
+  link: string;
+  active: boolean;
+  startTime?: Date[];
+  endTime?: Date[];
+  createdAt?: Date;
+  updatedAt?: Date;
+  ogImageUrl?: string;
+  ogTitle?: string;
+  ogDescription?: string;
+}
+
+interface PlutoBlogPostingState {
+  isLoading: boolean;
+  blogLink: string;
+  blogLinks: BlogLink[];
+}
+
+class PlutoBlogPosting extends React.PureComponent<{}, PlutoBlogPostingState> {
+  public constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      blogLink: "",
+      blogLinks: [],
+    };
+  }
+
+  public async componentDidMount() {
```

Comment:
You don't need async flag here if you don't wait for asynchronous function.

Reply:
Also, I think of that you need to get `paper` or `paperId` from props. Cuz if a user clicks the other paper link in the paper show page, PlutoBlogPosting component never be unmounted. (Because the parent component(PaperShow) never be unmounted.)
As a result, `componentDidMount` method doesn't be called again, and the blog post won't be changed at all.

If you get `paper` or `paperId` from `props`, you can detect the change of them in `componentWillReceiveProps` method.
Then you can call `this.handleClickReload()` method when `this.props.paper !== nextProps.paper`.